Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1070,12 +1070,13 @@ - (void)didReceiveData:(NSData *)data forTask:(NSURLSessionDataTask *)dataTask
         __weak typeof(self) weakSelf = self;
         [_concurrentOperationQueue pin_addOperationWithQueuePriority:PINRemoteImageManagerPriorityLow block:^{
             typeof(self) strongSelf = weakSelf;
-            PINImage *progressImage = [progressiveImage currentImageBlurred:shouldBlur maxProgressiveRenderSize:maxProgressiveRenderSize];
+            CGFloat imageProgress = 1.0;
```

Comment:
I'm not sure downloadProgress will be greater will it? This represents the download progress when the image was rendered I think?